Meta/Lagom/Tools/LibJSGCVerifier/src/CellsHandler.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <unordered_set>
+#include <clang/ASTMatchers/ASTMatchers.h>
+#include <clang/ASTMatchers/ASTMatchFinder.h>
+#include <clang/Tooling/Tooling.h>
+
+class CollectCellsHandler : public clang::tooling::SourceFileCallbacks
+                          , public clang::ast_matchers::MatchFinder::MatchCallback {
```

Comment:
looks weird, please follow the usual style please:

https://github.com/SerenityOS/serenity/blob/7c1455c3576c11f08ea7fc638b5106a11f9a7d0f/Userland/Libraries/LibWeb/Bindings/PlatformObject.h#L26-L28